<?xml version="1.0"?>

<launch>

	<!--Run the camera (requires usb_cam istalled)-->
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video2" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<!-- <param name="image_width" value="1280" /> -->
		<!-- <param name="image_height" value="960" /> -->
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
		<param name="camera_info_url" value="file://$(find mocapcam)/config/logitech_640_480.yaml"/>
		<!-- <param name="camera_info_url" value="file://$(find mocapcam)/config/logitech_1280_960.yaml"/> -->
	</node>

	<!--Use this to check if your camera is working-->
	<!--
	<node name="image_view" pkg="image_view" type="image_view" respawn="true" output="screen">
		<remap from="image" to="/usb_cam/image_raw"/>
		<param name="autosize" value="true" />
	</node>
	-->

</launch>